### ISSUE TITLE
Problems with graphviz / dot language

### ISSUE THREAD

**Issue by folixg** [vscode-icons/vscode-icons#2098]
- [x] I'm sure this issue is not a _duplicate_.

I'm currently working on a project for which I create graphviz/dot files. I realized that the files with `.dot` extension are shown as MS Word files. So far so good, since this is what i would expect from the [definition of the word type](https://github.com/vscode-icons/vscode-icons/blob/8138b5411f321d1d99f9b8b742f98c251b771c8e/src/iconsManifest/supportedExtensions.ts#L3512). I also managed to change the icon by adding the following snippet to my User Config:
```
   "vsicons.associations.files": [
        {
            "icon": "graphviz",
            "extensions": [
                "dot"
            ],
            "format": "FileFormat.svg"
        },
  ]
```

However, I would have expected the icon to change to the `graphviz` icon after opening the files, this did not happen (of course now I'm talking about the situation without the before mentioned update to my config).
[The `graphviz` icon is associated with the `languages.graphviz` language](https://github.com/vscode-icons/vscode-icons/blob/8138b5411f321d1d99f9b8b742f98c251b771c8e/src/iconsManifest/supportedExtensions.ts#L1429). However vscode has no built-in support for this language and [the extension I use](https://github.com/joaompinto/vscode-graphviz) associates the `languages.dot` language with the `.dot` files. 
![20190617-150425_scrot](https://user-images.githubusercontent.com/15685939/59606385-83df7c80-9100-11e9-971d-8f72bdde0d5d.png)
So I tried to adapt my config in order to get the `graphviz` icon assigned to `languages.dot` files:
```
    "vsicons.associations.files": [
        {
            "icon": "graphviz",
            "language": [
                "languages.dot"
            ],
            "format": "FileFormat.svg"
        },
  ]
```
Unfortunately this did not change the behaviour (yes, I applied the icon customization). My `.dot` files are still shown with a MS Word icon, even after I open one and it is identified as a `languages.dot` file.

So long story short, I have two questions:
1. Why is the language called `languages.graphviz`? I tried several extensions, they all seem to use `languages.dot` as identifier.
2. Why doesn't the association of `languages.dot` with the `graphviz` icon have any effect?

Please let me know if I can be of assistance in any way.

**Comment by JimiC** [comment]
1. `graphviz` is associated with the `dot` `language id`. No problem there.
2. https://github.com/vscode-icons/vscode-icons/wiki/FineTuning#some-examples pay attention to the `extends` and `overrides` attributes.

**Comment by JimiC** [comment]
Since you are using an extension that declares the `dot` `language id` and the `.dot` as an extension, you should see the `graphviz` icon next to a file with `.dot` file extension, without using any file association.

**Comment by JimiC** [comment]
I see now what you mean. Unfortunately, the way `vscode` works, any file`extension declaration (see `word` icon) overweights the `language id` declaration (see `graphviz`). Therefore, the proper way to address that issue is as you have shown, to disable the `word` icon, or furthermore to re-declare the `word` icon in `file association` with extensions that does not include `dot`, in case you want to keep the other declared file extensions of the `word` icon. In the later `overrides` is the attribute you will need.

**Comment by folixg** [comment]
Ok, I suspected something like that.
I will just disable the word icon since I don't have any word files in my project.
Thanks again.
